src/client/post/PostContent.js

```diff
@@ -207,7 +207,7 @@ class PostContent extends React.Component {
     const postMetaImage = postMetaData.image && postMetaData.image[0];
     const htmlBody = getHtml(body, {}, 'text');
     const bodyText = sanitize(htmlBody, { allowedTags: [] });
-    const desc = `${bodyText.substring(0, 140)} by ${author}`;
+    const desc = `${bodyText.substring(0, 140)}... by ${author}`;
```

Comment:
So it would be better if you created a helper function in one of our helper files that does string truncation. 

Right now if the bodyText is under 140 characters then it still adds an ellipsis, and that should not be the case.